WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Success card/MigrationSuccessCell.swift

```diff
@@ -0,0 +1,47 @@
+import UIKit
+
+@objc
+class MigrationSuccessCell: UITableViewCell {
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setup() {
+        let view = MigrationSuccessCardView() {
+            // TODO: add card presentation logic here
+        }
+        view.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(view)
+        contentView.pinSubviewToAllEdges(view)
+    }
+}
+
+class MigrationSuccessRow: ImmuTableRow {
+    var action: ImmuTableAction?
+
+    func configureCell(_ cell: UITableViewCell) {
+
+    }
+
+    static let cell = ImmuTableCell.class(MigrationSuccessCell.self)
+}
+
+extension BlogDetailsViewController {
+
+    @objc func migrationSuccesstSectionViewModel() -> BlogDetailsSection {
```

Comment:
In case this was a typo:

```suggestion
    @objc func migrationSuccessSectionViewModel() -> BlogDetailsSection {
```